app/scripts/modules/core/task/tasks.less

```diff
@@ -85,6 +85,11 @@
     }
   }
 
+  .task-comment {
+    padding-top: 5px;
+    padding-left: 40px;
+  }
+
```

Comment:
Instead of inlining the style above in the directive template, you can put it in here:
```less
task-comment {
  .row {
    margin-top: 10px;
    margin-bottom: 10px;
  }
}
```